x-pack/docs/en/security/authentication/configuring-saml-realm.asciidoc

```diff
@@ -0,0 +1,216 @@
+[role="xpack"]
+[[configuring-saml-realm]]
+=== Configuring a SAML realm
+
+The {stack} supports Security Assertion Markup Language Single Sign On (SAML SSO) 
+into {kib} with {es} as a backend service. In particular, the {stack} supports 
+the SAML 2.0 Web Browser SSO and the SAML 2.0 Single Logout profiles. It can 
+integrate with any identity provider (IdP) that supports at least the SAML 2.0 
+Web Browser SSO Profile. 
+
+In SAML terminology, the {stack} is operating as a _service provider_. For more 
+information, see {stack-ov}/saml-realm.html[SAML authentication] and 
+{stack-ov}/saml-guide.html[Configuring SAML SSO on the {stack}]. 
+
+[NOTE]
+--
+
+* If you configure a SAML realm for use in {kib}, you should also configure
+another realm, such as the native realm in your authentication chain.
+* These instructions assume that you have an existing identity provider.
+--
+
+To enable SAML authentication in {es} and add the {stack} as a service provider:
+
+. Enable SSL/TLS for HTTP. 
++
+--
+If your {es} cluster is operating in production mode, you must
+configure the HTTP interface to use TLS before you can enable SAML 
+authentication.
+
+See <<tls-http>>.
+--
+
+. Enable the Token Service. 
++
+--
+The {es} SAML implementation makes use of the {es} Token Service.  This service
+is automatically enabled if you configure TLS on the HTTP interface. You can 
+explicitly enable it by including the following setting in your 
+`elasticsearch.yml` file:
+
+[source, yaml]
+------------------------------------------------------------
+xpack.security.authc.token.enabled: true
+------------------------------------------------------------
+--
+
+. Generate a SAML metadata file. 
```

Comment:
I would suggest "Configure a SAML IdP metadata file"

"Generate" might lead people to use our saml-metadata utility which is not correct.

We've also had a couple of support cases of people not understanding the distinction between IdP metadata and SP metadata, so I think that it's worth being super clear here.

Reply:
Thanks @tvernum.  I think I must have assumed the same, since I was missing that step.  I've added it now, but I'm still uncertain which steps (if any) you can skip after importing that file. Can you help clarify?